framework-cloud/framework-cloud-common/src/main/java/org/kie/cloud/common/provider/KieServerClientProvider.java

```diff
@@ -33,6 +34,10 @@
     private static final long KIE_SERVER_TIMEOUT = 300_000L;
 
     public static KieServicesClient getKieServerClient(KieServerDeployment kieServerDeployment) {
+        return getKieServerClient(kieServerDeployment.getUrl(), kieServerDeployment);
+    }
+
+    public static KieServicesClient getKieServerClient(URL kieServerUrl, KieServerDeployment kieServerDeployment) {
```

Comment:
URL kieServerUrl is not used in this method. I think that this should be used in `KieServicesFactory.newRestConfiguration(kieServerUrl.toString(), kieServerDeployment.getUsername(), kieServerDeployment.getPassword(), KIE_SERVER_TIMEOUT);`

Reply:
Good catch, will fix it.